gen/DocumentFormat.OpenXml.Generator/Models/OfficeVersion.cs

```diff
@@ -11,4 +11,5 @@ public enum OfficeVersion
     Office2016 = 3,
     Office2019 = 4,
     Office2021 = 5,
+    Microsoft365 = 6,
```

Comment:
These numbers don't matter except Office2007=0. Can you remove the others?

Reply:
I thought you added those. Must have been a test I did and forgot to remove. Yes, I'll remove them.

Reply:
I did, but realizing now that the numbers themselves don't matter except the `default(OfficeVersion)` value